test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -670,7 +670,7 @@ withBlockHeaderDb
 withBlockHeaderDb iordb b = withResource start stop
   where
     start = do
-        rdb <- iordb
+        rdb <- testRocksDb "withBlockHeaderDb" =<< iordb
```

Comment:
It creates a new unique namespace on the rocks db. That way the rocks db can be uses concurrently by more than one block headers dbs. Which supports to running tests in parallel without risk of interference.

(There is a chance that concurrent use works fine without this, since the blocks are content addressed by their hash, but that's not guaranteed and also performance could be worse.)

Reply:
ok